### ISSUE TITLE
AmiGO public-facing documentation should not be on the wiki, should be internal in repo

### ISSUE THREAD

**Issue by paolaroncaglia** [geneontology/amigo#297]
Hi,

1) This may have been reported already (but not among latest issues). If so, please link to existing ticket(s) and/or mark as duplicate.

2) There is an upstream issue which is the somewhat unfit state of AmiGO documentation. 

3) Related to 2) Is there a designated person to take care of AmiGO documentation, or is it the same person responsible for software issues?

4) Currently, users visiting AmiGO2 http://amigo.geneontology.org/amigo, and wishing to look for help from the top menu, are redirected to the wiki: http://wiki.geneontology.org/index.php/AmiGO_2_Manual:_Home
This is confusing and inappropriate (as we resolved that the wiki is for internal use only, and we need to move away from it in public display). Note that 35% of polled GO users access GO annotations or ontology via AmiGO (source: GO user survey 2015).

Summing up: Documentation for AmiGO2 needs to be improved and hosted on the website. Please. :-)
This would be useful for the GO tools registry too; currently, I left the 'documentation link' slot empty so we don't point to the wiki publicly. Ultimately we want to link the registry on the GO website.

Thanks,

Paola

**Comment by kltm** [comment]
Well, a bit more story here, but the short of it is that at a point in time it was decided that the AmiGO documentation *should* be in the wiki so that 1) the community of biologists could support it in a way that makes sense to them and that 2) it was not the developers' problem keeping it up to date. In practice, however, the result was less community involvement than expected. If it were to all be done over starting right now, a better way would probably be to have all the docs kept as versionable markdown files in the repo, but train a set of people to be able to update them. Because of the versioning issues, I do not think it's appropriate to have the documentation live directly in the go-site drupal (although there are sensible arguments for it).

With that said:
I'm not sure which issue you're referring to or the difference between 1 and 2. For 3: no there is not; any anybody with wiki access can fix things, although forwarding must be preserved.

**Comment by cmungall** [comment]
Wiki documentation is better than none, but it is not a good long term solution. Same with drupal.

The documentation should be integrated into the web interface as much as possible. There should be links out to ontology concepts that are application neutral. For example, we should have a general page that attractively shows how gene products propagate up the graph. There could be mention of how it looks in amigo, or any other browser, but the page would largely be about the concepts not the tool. This points to a good division of labor between developers and other members of the GOC. E.g. the ontology group could contribute the conceptual page, which should be on the main website.

Bear in mind that in various parts of amigo additional documentation may be necessary because of known issues, such as the underlying architecture being exposed too much. With a more intuitive flow, the need for external docs is lessened. This observation doesn't help much in the short term, however.

We need to be forward-thinking here as well. We will be introducing new concepts to the user when lego models are visible in amigo. Again, we need a minimal amount of inline documentation, but the key will be good quality documentation that explains the concepts.

Another thing that would help here would be videos

**Comment by kltm** [comment]
I'd say internal markdown in the repo, so we're version same, but pathed so that others (like planteome) could just drop theirs in. Videos would be linked externally, but that would be flexible with the different paths.

**Comment by paolaroncaglia** [comment]
Thanks both for your comments.

Seth, to clarify this:
“I'm not sure which issue you're referring to or the difference between 1 and 2.”

The main issue I wished to report was AmiGO documentation being on the wiki rather than the website. 
My point 1) “This may have been reported already (but not among latest issues). If so, please link to existing ticket(s) and/or mark as duplicate.” refers to - maybe the matter of AmiGO documentation being on the wiki rather than the website was discussed already, and maybe a GH ticket on this tracker was already created in this regard, but I couldn’t find it. Based on what you write, there probably is no other ticket. So 1) is dealt with :-)

My point 2) “There is an upstream issue which is the somewhat unfit state of AmiGO documentation.” refers to - the documentation is on the wiki and we may prefer to have it on the website instead, but before we can do that, current documentation may need to be improved. As said, currently users visiting AmiGO2 http://amigo.geneontology.org/amigo, and wishing to look for help from the top menu, are redirected to the wiki here: http://wiki.geneontology.org/index.php/AmiGO_2_Manual:_Home. Looking at this page and trying to find help on a given subject supposing you’re a newbie, doesn’t feel entirely straightforward to me, but thanks to your explanation I can see the rationale behind the documentation being fragmented and hosted as is. However, one point I think we’d all agree is confusing, is if you look at the bottom of the (short) front wiki, there are two links:
AmiGO 2 Manual (http://wiki.geneontology.org/index.php/Category:AmiGO_2_Manual)
AmiGO 2 (http://wiki.geneontology.org/index.php/Category:AmiGO_2)
If our users are supposed to go looking for documentation starting from one of those, I bet they wouldn’t think along the lines of our wiki categories system (and therefore see that the links contents are all the same, but would rather wonder what the difference is between the two ‘pages’ (“Is the content of the links exactly the same? What is the right starting point?”).

Also, there is a ‘telling sign’ in one of the main sub-pages that there was a plan to continue work on documentation:

http://wiki.geneontology.org/index.php/AmiGO_2_Manual:_Overview#Introduction
“While we are getting the new documentation sorted out, plase take a look at the pages in Category:AmiGO_2_Manual.”

I hope this clarifies. Thanks,

Paola

**Comment by kltm** [comment]
This has been taken over by #371 .
